Lib/test/test_compileall.py

```diff
@@ -18,7 +18,7 @@
     # compileall relies on ProcessPoolExecutor if ProcessPoolExecutor exists
     # and it can function.
     from multiprocessing.util import _cleanup_tests as multiprocessing_cleanup_tests
-    from concurrent.futures import ProcessPoolExecutor
+    from concurrent.futures import ProcessPoolExecutor  # noqa: F401
```

Comment:
Why `ProcessPoolExecutor` is needed?

Reply:
It's explained in the comment, 2 lines above. It's a check to define `_have_multiprocessing`.